src/content/docs/en/setup/bioskeys.mdx

```diff
@@ -8,33 +8,38 @@ These are the keys (or combination of keys) used to enter the boot menu or UEFI/
-| Microsoft              | Volume Up+Power           | -                          | There is a boot menu available in the UEFI Settings                                                                                                               |
+| Microsoft (Surface)    | Hold Volume Up            | -                          | There is a boot menu available in the UEFI Settings. Volume Up should be held after a reboot is initiated.                                                        |
 | Origin PC              | F2                        | F2                         |                                                                                                                                                                   |
+| Panasonic (Toughbook)  | F2                        | Del                        | There is a boot menu in the 'exit' screen of the bios.                                                                                                            |
+| Panasonic (Let's Note) | F2, Del                   | Enter                      |                                                                                                                                                                   |
 | Razer                  | Del, F1                   | Esc, F12                   |                                                                                                                                                                   |
 | Samsung                | F2                        | F10                        |                                                                                                                                                                   |
+| Sony (Vaio)            | F1, F2, F3, Assist        | esc, F10, F11, Assist      | Use the Assist Button when the computer is fully off, not in the process of booting.                                                                              |
 | Supermicro             | Del, F4, F11              | F11                        |                                                                                                                                                                   |
 | System76               | F2                        | F7                         |                                                                                                                                                                   |
 | Toshiba                | F2                        | F12                        |                                                                                                                                                                   |
+| Sharp (Dynabook)       | F2                        | F12                        | Some Toshibas will be branded as Dynabook, this is because Sharp recently purchased the brand. The bios keys are the same.                                        |
```

Comment:
```suggestion
| Sharp (Dynabook)       | F2                        | F12                        | Some Toshibas will be branded as Dynabook, this is because Sharp recently purchased the brand. The BIOS keys are the same.                                        |
```